web/client/product/components/viewer/MapViewerCmp.jsx

```diff
@@ -36,28 +37,41 @@ class MapViewerComponent extends React.Component {
             params: {}
         }
     };
+
+    state = {
+        initMap: null
+    };
+
     UNSAFE_componentWillMount() {
         const id = this.props.match.params.mapId || '0';
         const contextId = this.props.match.params.contextId;
-        this.updateMap(id, contextId);
+        this.setState({ initMap: () => {
+            this.updateMap(id, contextId);
+            this.setState({initMap: null});
+        }});
```

Comment:
Based on the pvt discussion we should do the follow:
- revert this fix for the moment with all related code around
- plugin that has still issue because of this should be kept static for the moment
- once we have the list with all the problematic plugins we can evaluate how to proceed

Reply:
Rolled back, list of plugins is to be added in the comment down below in PR.